.github/scripts/spacecheck.pl

```diff
@@ -92,41 +84,49 @@ sub eol_detect {
 
     my @err = ();
 
-    if(!fn_match($filename, @tabs) &&
-        $content =~ /\t/) {
+    if(!fn_match($filename, @tabs)
+        && $content =~ /\t/)
+    {
         push @err, "content: has tab";
     }
 
     my $eol = eol_detect($content);
 
-    if($eol eq "" &&
-        !fn_match($filename, @mixed_eol)) {
+    if($eol eq ""
+        && !fn_match($filename, @mixed_eol))
+    {
         push @err, "content: has mixed EOL types";
     }
 
-    if($eol ne "crlf" &&
-        fn_match($filename, @need_crlf)) {
+    if($eol ne "crlf"
+        && fn_match($filename, @need_crlf))
+    {
         push @err, "content: must use CRLF EOL for this file type";
     }
 
-    if($eol ne "lf" && $content ne "" &&
-        !fn_match($filename, @need_crlf) &&
-        !fn_match($filename, @mixed_eol)) {
+    if(    $eol ne "lf"
+        && $content ne ""
+        && !fn_match($filename, @need_crlf)
+        && !fn_match($filename, @mixed_eol))
```

Comment:
Knuth or not, this was not used in curl, and rarely seeing this anywere else also. This doesn't seem like an improvement.